app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -338,6 +338,7 @@ def remove_content
     api :POST, "/repositories/:id/upload_content", N_("Upload content into the repository")
     param :id, :number, :required => true, :desc => N_("repository ID")
     param :content, File, :required => true, :desc => N_("Content files to upload. Can be a single file or array of files.")
+    param :content_type, String, :required => false, :desc => N_("srpm, drpm, rpm (default)")
```

Comment:
We probably want a way to look up the examples here automatically, probably using the RepsitoryTypeManager, although you may need to add some functionality there

Reply:
Ah, yes... the description was just 'filler' here, but that's wise. However, isn't RepositoryTypeManager things like 'yum', 'docker', etc? I thought we were not going to create srpm/drpm repo type since they are 'yum'.

Reply:
oh and i already mentioned that here :)